lib/worker.js

```diff
@@ -164,7 +164,7 @@ Worker.prototype._start = function() {
         self.serviceReturns = res;
         // Make sure that only JSON-serializable values are returned.
         try {
-            return [JSON.parse(JSON.stringify(res))];
+            return JSON.parse(JSON.stringify(res));
```

Comment:
I think it makes more sense to return it this way, and then in master near

    if (self.config.num_workers === 0) {
        // No workers needed, run worker code directly.
        // FIXME: Create a Master / Worker dynamically on _start()?
        return Worker.prototype._start.call(self);
    }

wrap that in an array.

If you really wanted to match master as it currently is, it should be `[JSON.parse(JSON.stringify(res))[0]]` but see below.

Also, what about the error case here.  That's just `[e]`.